ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -57,11 +57,20 @@ message Commands {
   // Required
   repeated Command commands = 8;
 
-  // The length of the time window during which all commands with the same ``act_as`` parties and
-  // the same command ID will be deduplicated.
-  // Duplicate commands submitted before the end of this window return an ALREADY_EXISTS error.
-  // Optional
-  google.protobuf.Duration deduplication_time = 9;
+  oneof deduplication_period {
+
+      // Specifies the length of the deduplication period.
+      // It is interpreted relative to the local clock at some point during the submission's processing.
+      //
+      // Must be non-negative.
+      google.protobuf.Duration deduplication_time = 9;
```

Comment:
Did we agree or not on deprecating this field, and introducing a new one, to force people to understand the new "looking back" semantics?

Happy to do it in a separate PR, but I don't want to drop it.

Reply:
Yes, adding a new one in a different PR because this one is too big already so I'm trying to limit the changes I introduce.